recipes/sdformat-mjcf/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "sdformat-mjcf" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gazebosim/gz-mujoco/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a0bd5e4f5eba54824ee7c09ce64d87f930f805d4ebc6f7fc8baa768c0bde3b28
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install ./sdformat_mjcf -vv --no-deps
+  number: 0
+  entry_points:
+    - sdf2mjcf = sdformat_mjcf.sdformat_to_mjcf.cli:main
+    - mjcf2sdf = sdformat_mjcf.mjcf_to_sdformat.cli:main
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - wheel
+    - setuptools
+  run:
+    - python >=3.7
+    - gz-math7-python
+    - sdformat13-python
+    - dm-control
+
+test:
+  imports:
```

If you want to keep the test files around, you can use [`source_files`](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#source-files):
```suggestion
test:
  source_files:
    - sdformat_mjcf/tests
  imports:
```